leftwm/src/bin/leftwm-check.rs

```diff
@@ -36,7 +36,7 @@ async fn main() -> Result<()> {
     );
     println!(
         "\x1b[0;94m::\x1b[0m LeftWM git hash: {}",
-        git_version::git_version!(fallback = option_env!("GIT_HASH").unwrap_or("NONE"))
+        git_version::git_version!(fallback = "unknown")
```

Comment:
Out of curiosity, did this get deprecated?

Reply:
Yeah, I believe the upstream repo fixed the depenency to the now deprecated `proc-macro-hack`.

Do you think we should add some tests to verify we also can get the `unknow` output releably? (I guess we should do this in a separate PR then, though.)

Reply:
After thinking of this, I don't have an idea how to spoof the `git hash` or the absence of one nicely for a test. So I guess we just keep it as is for now.